### ISSUE TITLE
Windows 10 x64 Notebook - Performous 1.1 is not starting

### ISSUE THREAD

**Issue by Stefan1200-de** [performous/performous#292]
After executing the Performous.exe I only see that the Vocaluxe process is taking around 32% for an endless time (waited around 10 minutes until I took the Task Manager). This error also occurs with an empty songs directory. infolog.txt (debug log level) is attached to this issue.
[infolog.txt](https://github.com/performous/performous/files/1352288/infolog.txt)

On my Windows 10 x64 desktop PC everything is working fine with the same USB microphone. Tested this with only one capture device connected to my notebook (the USB microphone). It is more or less the same software installed as on my desktop PC.

OS: Windows 10 x64
CPU: Intel i5 5200U
RAM: 8 GByte RAM
GPU: NVidia 940M & Intel HD (multi GPU device!)

What can I do to help you finding this issue? By the way: Vocaluxe 0.4.0 has the same issue, while UltraStar Deluxe 2017.8.0 is working fine.

**Comment by Baklap4** [comment]
What model of Intel hd graphics do you have? We are aware of the bug related to the 1.1 tag and some Intel HD graphics card... #174

**Comment by Stefan1200-de** [comment]
Intel HD Graphics 5500

Forcing Performous to use the NVidia 940M GPU does not help.

**Comment by nieknooijens** [comment]
> Forcing Performous to use the NVidia 940M GPU does not help.

well... then it's something todo with windows and how it handles openGL.
stupid question: have you got the latest nvidia drivers from nvidia itself?
the reason I'm asking this is because I encountered some laptops in the past that had an nvidia driver from the laptop manufacturer installed, which was only a partial driver (only including directX and NO openGL!)

**Comment by Stefan1200-de** [comment]
Updated the NVidia driver to 385.69 (downloaded today from NVidia website). The Intel HD Graphics 5500 driver was already up to date. Tried to start with forcing to NVidia GPU and with default settings (which means Intel GPU), same result: Performous still don't start.

**Comment by Lord-Kamina** [comment]
I don't think this is the same graphics issue. That bug didn't result in any problems starting performous itself. It just resulted in distorted graphics. Further, I've tried performous on W10 x64 both with some models of intel hd without any issue (the bug appears to be related to haswell-broadwell, _maybe_ skylake) gpus and also both AMD and NVidia cards without any issues: All the way from an HD6770 to an R9 290 and from a GT 750 to a GTX 1050, respectively.

We need more info. Screenshots and a debug log at least. Try

```
set FC_DEBUG=1024
"%programfiles(x86)%\Performous.exe" --log debug 1>LOG.txt 2>&1
```

And attach that log.

Also, please try this build and report whether there's any improvement: 

https://www.dropbox.com/s/ols04zbjuen7v0o/Performous-1.1-318-g9a1b15ae.exe?dl=0

**Comment by Stefan1200-de** [comment]
First: Release Version 1.1
[LOG.txt](https://github.com/performous/performous/files/1354762/LOG.txt)
[infolog.txt](https://github.com/performous/performous/files/1354763/infolog.txt)
Screenshot is not possible, since there is no Performous window coming up, it is just visible in task manager eating around 32% of my CPU in an endless loop.

Second: Your test build, **which is running by the way.**
[LOG.txt](https://github.com/performous/performous/files/1354768/LOG.txt)
[infolog.txt](https://github.com/performous/performous/files/1354766/infolog.txt)

**Comment by Baklap4** [comment]
@Stefan1200-de i see on your second note you're running on your nvidia card, can you try forcing the intel card?

**Comment by nieknooijens** [comment]
added fix comitted since @Lord-Kamina 's build apparently solves it

**Comment by Stefan1200-de** [comment]
@Baklap4 Well, your test build is starting with both GPUs, but on my Intel HD Graphics 5500 GPU it looks weird:
![2017-10-04 21_39_49-performous 1 1-318-g9a1b15ae](https://user-images.githubusercontent.com/15168351/31196236-49abab02-a94d-11e7-86d9-90c7370d96d7.png)

**Comment by Baklap4** [comment]
@Stefan1200-de that's a bug currently tracked by #174

**Comment by Lord-Kamina** [comment]
Well, at least now I know that that didn't fix THAT issue hahahaha.

**Comment by Baklap4** [comment]
@Lord-Kamina sad but true yes :( however something did fix the 1.1 build from not starting; so that's Progress!

**Comment by Baklap4** [comment]
Closed as duplicate of #174 and label: 'fix committed' @Stefan1200-de Let's move to that thread for further investigation
